src/utils/getCropLifecycleDuration.js

```diff
@@ -1,17 +1,11 @@
 import { memoize } from './memoize'
 
-// TODO: Refactor this to accept just a plain cropTimetable
-// https://github.com/jeremyckahn/farmhand/issues/415
 /**
- * @param {{ cropTimetable: Object }} crop
+ * @param {{ cropTimeline: Array}} crop
```

Comment:
This type could be a little narrower:

```suggestion
 * @param {{ cropTimeline: number[] }} crop
```